assets/user-profile/components/EditNotificationScheduleModal.tsx

```diff
@@ -52,6 +53,8 @@ class EditNotificationScheduleModalComponent extends React.Component<IProps, ISt
 
         this.onSubmitForm = this.onSubmitForm.bind(this);
         this.formRef = React.createRef<HTMLFormElement>();
+
+        this.TIME_FORMAT = TIME_FORMAT.split(':')[0] === 'HH' ? '24-hours' : '12-hours';
```

Comment:
would it work to move this as a constant at the top of the file instead to being specific to a class instance?

Reply:
yes.. and I will do it